client-service/src/main/java/com/telecominfraproject/wlan/client/controller/ClientController.java

```diff
@@ -397,10 +406,19 @@ public ListOfClientSessions updateSessionsBulk(@RequestBody List<ClientSession>
     		//nothing to do here
     		return new ListOfClientSessions();
     	}
-    	
 
         LOG.debug("Updating Client sessions {}", clientSessions);
         
+        try {
+            for (ClientSession clientSession : clientSessions)
+            {
+                clientSessionValidator.validateClientSession(clientSession);
+            }
+        } catch (ClientSessionValidatorException e) {
+            LOG.error(e.getMessage());
+            throw new DsDataValidationException(e.getMessage());
```

Comment:
It might be helpful to add an identifier here, e.g. macAddress, since it is a bulk update. Either the LOG statement or the exception message